pkgs/servers/pulseaudio/default.nix

```diff
@@ -154,11 +154,7 @@ stdenv.mkDerivation rec {
   '';
 
   passthru = {
-    pulseDir =
-      if (lib.versionAtLeast version "16.0") then
-        "lib/pulseaudio"
-      else
-        "lib/pulse-" + lib.versions.majorMinor version;
+    pulseDir = "lib/pulseaudio";
```

Comment:
Thats only used in two places where we can now hardcode the path and remove this.

Reply:
Done.